.github/workflows/unit.yml

```diff
@@ -58,6 +58,9 @@ jobs:
 
     - run: yarn test-proto # Run before unit-core because the roundtrip json is needed for proto tests.
 
+    - run: yarn build-viewer && yarn mocha --testMatch viewer/test/viewer-test-pptr.js -t 'proto roundtrip report'
```

Comment:
Why don't we just run viewer tests as part of the unit CI job and remove from basics?